deployment/data-streams/jobs/llo.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/pelletier/go-toml/v2"
 )
 
-type DonJobSpec struct {
+type LLOJobSpec struct {
```

Comment:
I renamed "Don" to "LLO" in order to reduce the confusion caused by referring to this job as a "DON job".